packages/api/src/constants.js

```diff
@@ -1,4 +1,36 @@
 // let MAGIC_SECRET_KEY, SALT, PINATA_JWT, SENTRY_DSN, DATABASE_TOKEN, CLUSTER_SERVICE, LOGTAIL_TOKEN, MAILCHIMP_API_KEY, METAPLEX_AUTH_TOKEN
+import { config } from 'dotenv'
+
+config({ debug: true })
+
+const {
+  MAGIC_SECRET_KEY,
+  SALT,
+  SENTRY_DSN,
+  DATABASE_TOKEN,
+  CLUSTER_SERVICE,
+  LOGTAIL_TOKEN,
+  MAILCHIMP_API_KEY,
+  CLUSTER_API_URL,
+  DATABASE_URL,
+  DEBUG,
+} = process.env
+
+if (!SALT) {
```

Comment:
Accessing any of these globals when they are undefined will throw an error so we already fail fast.

This is also why you see `typeof CLUSTER_BASIC_AUTH_TOKEN !== 'undefined'` in this file for example, because this var is optional.

Reply:
Is this new(ish)? I ended up doing this because I was missing env vars, and had to dig around trying to find what was missing.

That said, we'd have to go a different route if we're not using process.env. 
I'll see if I hit it again, and open a new PR if that happens.